config/deploy.rb

```diff
@@ -22,8 +22,7 @@
 # set :pty, true
 
 # Default value for :linked_files is []
-# set :linked_files, %w[config/database.yml]
-set :linked_files, %w[config/honeybadger.yml config/rgeoserver.yml config/rgeoserver_public.yml config/rgeoserver_restricted.yml tmp/resque-pool.lock]
```

Comment:
i'm pretty unfamiliar with this codebase, so might be a naive question, but no need for `tmp/resque-pool.lock` anymore?

Reply:
Thanks sorry I missed that on a merge conflict. Added back

Reply:
thanks!